packages/flutter/lib/src/widgets/visibility.dart

```diff
@@ -0,0 +1,209 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'ticker_provider.dart';
+
+/// Whether to show or hide a child.
+///
+/// By default, the [visible] property controls whether the [child] is included
+/// in the subtree or not; when it is not [visible], the [replacement] child
+/// (typically a zero-sized box) is included instead.
+///
+/// A variety of flags can be used to tweak exactly how the child is hidden.
+///
+/// These widgets provide some of the facets of this one:
+///
+///  * [Opacity], which can stop its child from being painted.
+///  * [Offstage], which can stop its child from being laid out or painted.
+///  * [TickerMode], which can stop its child from being animated.
+///  * [ExcludeSemantics], which can hide the child from accessibility tools.
+///  * [IgnorePointer], which can disable touch interactions with the child.
+///
+/// Using this widget is not necessary to hide children. The simplest way to
+/// hide a child is just to not include it, or, if a child _must_ be given (e.g.
+/// because this is a [StatelessWidget]) then to use [SizedBox.shrink] instead
```

Comment:
the 'this' in this sentence is not clear anymore. I think you're now referring a new hypothetical parent instead of this widget specifically.

Reply:
Good call. Fixed.